scripts/generate_aviti_run_manifest.py

```diff
@@ -0,0 +1,323 @@
+#!/usr/bin/env python
+
+import logging
+import os
+import re
+import shutil
+from argparse import ArgumentParser, Namespace
+from datetime import datetime as dt
+
+import pandas as pd
+from genologics.config import BASEURI, PASSWORD, USERNAME
+from genologics.entities import Process
+from genologics.lims import Lims
+from Levenshtein import hamming as distance
+
+from scilifelab_epps.epp import upload_file
+from scilifelab_epps.wrapper import epp_decorator
+from scripts.generate_minknow_samplesheet import get_pool_sample_label_mapping
+
+TIMESTAMP = dt.now().strftime("%y%m%d_%H%M%S")
+LABEL_SEQ_SUBSTRING = re.compile(r"[ACGT]{4,}(-[ACGT]{4,})?")
+
+
+def get_runValues_section(process: Process, file_name: str) -> str:
+    """Generate the [RUNVALUES] section of the AVITI run manifest and return it as a string."""
+
+    read_recipe = "-".join(
+        [
+            str(process.udf.get("Read 1 Cycles", 0)),
+            str(process.udf.get("Index Read 1", 0)),
+            str(process.udf.get("Index Read 2", 0)),
+            str(process.udf.get("Read 2 Cycles", 0)),
+        ]
+    )
+
+    runValues_section = "\n".join(
+        [
+            "[RUNVALUES]",
+            "KeyName, Value",
+            f"lims_step_name, {safe_string(process.type.name)}",
+            f"lims_step_id, {process.id}",
+            f"lims_step_operator, {process.technician.name}",
+            f"file_name, {safe_string(file_name)}",
+            f"file_timestamp, {TIMESTAMP}",
+            f"read_recipe, {read_recipe}",
+        ]
+    )
+
+    return runValues_section
+
+
+def get_settings_section() -> str:
+    """Generate the [SETTINGS] section of the AVITI run manifest and return it as a string."""
+    settings_section = "\n".join(
+        [
+            "[SETTINGS]",
+            "SettingName, Value",
+        ]
+    )
+
+    return settings_section
+
+
+def get_samples_section(process: Process) -> str:
+    """Generate the [SAMPLES] section of the AVITI run manifest and return it as a string."""
+
+    phix_loaded: bool = process.udf["PhiX Loaded"]
+
+    # Get the analytes placed into the flowcell
+    arts_out = [op for op in process.all_outputs() if op.type == "Analyte"]
+    lanes = [art_out.location[1].split(":")[1] for art_out in arts_out]
+
+    # If only a single pool is added to the LIMS container, treat it as though it was loaded into both lanes
```

Comment:
Hi Alfred, we defined the AVITI container as a two-well container. There will always be two lanes and the lab ppl will need to create replicates when loading them.